code/modules/surgery/organs/organ_external.dm

```diff
@@ -638,6 +652,23 @@ Note that amputating the affected organ does in fact remove the infection from t
 		owner.handle_splints()
 	return TRUE
 
+/obj/item/organ/external/proc/severe_burn()
+	if(status & ORGAN_BURNT || is_robotic() || cannot_burn)
+		return
+	if(owner)
+		owner.visible_message(
+			"<span class='warning'>You hear a quiet sizzling sound coming from \the [owner].</span>",
```

Comment:
```suggestion
			"<span class='warning'>You hear a quiet sizzling sound coming from [owner]'s [name].</span>",
```
It would be better to make it clear who's organ is being sizzled, assuming `owner` is the persons name and `name` is the limb name

Reply:
That would be a correct assumption. I'll get these done.